controllers/user.js

```diff
@@ -1,15 +1,36 @@
 const User = require('../models/user');
+const Exercise = require('../models/workoutlog');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7828699)
```suggestion
const Exercise = require("../models/workoutlog");
```